packages/nimble-components/src/table-column/number-text/models/tests/decimal-formatter.spec.ts

```diff
@@ -120,6 +120,7 @@ describe('DecimalFormatter', () => {
                 () => {
                     const formatter = new DecimalFormatter(
                         locale,
```

Comment:
I think we should have test coverage in this file for the case where min and max digits are different.